gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EventController.groovy

```diff
@@ -32,4 +32,9 @@ class EventController {
   void webhooks(@PathVariable("type") String type, @PathVariable("source") String source, @RequestBody Map event) {
     eventService.webhooks(type, source, event)
   }
+
+  @RequestMapping(value = "/webhooks/{type}", method = RequestMethod.POST)
+  void webhooks(@PathVariable("type") String type, @RequestBody Map event) {
+    eventService.webhooks(type, event)
```

Comment:
You still haven't provided a compelling reason why we need to maintain two sets of endpoints instead of the teamcity endpoint posting to /webhooks/ci/teamcity.

This PR adds unnecessary endpoints and I would rather this method and the corresponding one in echo be removed.